sdk/src/main/java/com/mapbox/maps/MapController.kt

```diff
@@ -265,9 +265,10 @@ internal class MapController : MapPluginProviderDelegate, MapControllable {
       val logoPluginClass = Class.forName(PLUGIN_LOGO_CLASS_NAME) as Class<LogoPlugin>
       createPlugin(mapView, logoPluginClass)
     } catch (ex: ClassNotFoundException) {
-      Logger.d(
+      Logger.w(
         TAG,
-        "The Mapbox logo wordmark must remain enabled in accordance with our Terms of Service. See https://www.mapbox.com/legal/tos for more details."
+        mapView?.context?.getString(R.string.mapbox_warning_logo_disabled)
```

Comment:
We can use the context instance in `mapInitOptions` here, it's not nullable.

Reply:
Very nice, thanks!